app/assets/javascripts/admin/admin_ofn.js.coffee

```diff
@@ -12,5 +12,6 @@ angular.module("ofn.admin", [
   "admin.orders"
 ]).config ($httpProvider, $locationProvider, $qProvider) ->
   $httpProvider.defaults.headers.common["Accept"] = "application/json, text/javascript, */*"
+  # for the next line, you should also probably check file: app/assets/javascripts/admin/utils/utils.js.coffee
   $locationProvider.hashPrefix('')
   $qProvider.errorOnUnhandledRejections(false)
```

Comment:
Has the fix been merged independently? This commit is just adding comments now.

Reply:
Yes, the comment is useful but I would change the commit message to reflect what's happening.